Rename nodelink to MultiLink (#119)
multinet-app/view-nodelink: README.md

# MultiLink

This is the MultiLink data visualization app for the Multinet project.

> It is a typescript node-link application, written using the Vue.js framework,

Reviewer: ```suggestion
It is an interactive node-link diagram, written in TypeScript using the Vue.js framework,
```
